### PR TITLE
Profile fragment

### PR DESCRIPTION
**Pull request by olaokoroafor**
Description:
 - User can actuallly make friends on the app
 - User can follow people from their profiles as well as from the find friends search

### CODE REVIEW

File: FoodFriends/app/src/main/java/com/example/foodfriends/adapters/FindFriendsAdapter.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                Glide.with(context)
+                        .load(R.drawable.ic_baseline_person_add_24)
+                        .into(ivAddFriend);
+            }
+
+            ivAddFriend.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    if(follows){
+                        //do something to database
```
Comment (asaxena1312) [inline FoodFriends/app/src/main/java/com/example/foodfriends/adapters/FindFriendsAdapter.java]: can you be more specific in terms of what you want to do the database? Like add/remove something

File: FoodFriends/app/src/main/java/com/example/foodfriends/adapters/FindFriendsAdapter.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            ivAddFriend = itemView.findViewById(R.id.ivAddFriend);
+            tvUsername = itemView.findViewById(R.id.tvSearchUsername);
+            tvLocation = itemView.findViewById(R.id.tvSearchLocation);
+            itemView.setOnClickListener(this);
+        }
+
+        @Override
+        public void onClick(View v) {
+            /*
+
```
[the remark this answers is not in the document]
Comment (olaokoroafor) [reply FoodFriends/app/src/main/java/com/example/foodfriends/adapters/FindFriendsAdapter.java:89]: deleted!

File: FoodFriends/app/src/main/java/com/example/foodfriends/adapters/FindFriendsAdapter.java [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                // create intent for the new activity
+                Intent intent = new Intent(context, PostDetailActivity.class);
+                intent.putExtra(Post.class.getSimpleName(), post);
+                context.startActivity(intent);
+            }*/
+
+        }
+
+        public void bind(ParseUser user) {
+            tvUsername.setText(user.getUsername());
```
[the remark this answers is not in the document]
Comment (olaokoroafor) [reply FoodFriends/app/src/main/java/com/example/foodfriends/adapters/FindFriendsAdapter.java:105]: Username cannot be null or blank!

File: FoodFriends/app/src/main/java/com/example/foodfriends/adapters/FindFriendsAdapter.java (hunk 1 above)
Comment (olaokoroafor) [reply FoodFriends/app/src/main/java/com/example/foodfriends/adapters/FindFriendsAdapter.java]: removed  this comment!

**Pull request merged**